ebean-migration/src/test/resources/dbmig/1.1__initial.sql

```diff
@@ -1,3 +1,5 @@
 create table m1 (id integer, acol varchar(20));
-
+-- Check with DB2:
+-- call sysproc.admin_cmd('reorg table m1');
```

Comment:
When the reorg is present, all locks of the current connection are dropped and the other waiting thread will continue.